lib/rules/require-line-break-after-variable-assignment.js

```diff
@@ -61,6 +61,12 @@ module.exports.prototype = {
     check: function(file, errors) {
         var lastDeclaration;
         file.iterateNodesByType('VariableDeclaration', function(node) {
+            if (node.parentNode.type === 'ForStatement' ||
```

Comment:
Should I change to ` if (node.parentNode.type &&  node.parentNode.type === ...)`?